test/fs/test_open_wasmfs.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <emscripten/emscripten.h>
+#include <stdio.h>
+
+
+int main() {
+  EM_ASM(
+    FS.writeFile('testfile', 'a=1\nb=2\n'); // writeFile already works with WasmFS
```

Comment:
```suggestion
    FS.writeFile('testfile', 'a=1\nb=2\n');
```
(that comment helps read this PR, but it doesn't seem like it would be useful after this PR lands, so it could've been in the PR as a comment and not in the source code)

Reply:
I've removed this comment. I'll make sure to keep this in mind going forward!